test/onnx/test_op_consistency.py

```diff
@@ -246,7 +246,7 @@ def _get_test_class_name(cls, num, params_dict) -> str:
             "name": f"TestOnnxModelOutputConsistency_opset{opset}",
             "opset_version": opset,
         }
-        for opset in onnx_test_common.TESTED_OPSETS
+        for opset in onnx_test_common.ONNXRT_SUPPORTED_OPSETS
```

Comment:
```suggestion
        for opset in onnx_test_common.TESTED_OPSETS
```

To test up to opset 17